commands/command_pre_push.go

```diff
@@ -55,7 +54,13 @@ func prePushCommand(cmd *cobra.Command, args []string) {
 	}
 	lfs.Config.CurrentRemote = args[0]
 
-	uploadedOids := lfs.NewStringSet()
+	cli := newClient()
+	cli.RemoteName = lfs.Config.CurrentRemote
```

Comment:
Hmm, feels weird to have to copy the remote name when everything else uses Config directly, feels like this might be fragile - either forgetting to assign it, or other code which changes lfs.Config.CurrentRemote thinking it applies everywhere (some tests do this already I think). I think we should keep a single source of truth.